src/filecache.c

```diff
@@ -752,6 +752,10 @@ static void get_fresh_fd(filecache_t *cache,
          *  Not sure how to remediate without doing way more work and
          *  introducing way more problems than the fix will fix.
          */
+        const float samplerate = 1.0; // always sample stat
+        stats_counter("unexpected-not-found", 1, samplerate);
+        log_print(LOG_WARNING, SECTION_FUSEDAV_STAT, "Encountered an unexpected 404/410: Repsonse code: %u: ", response_code);
```

Comment:
```suggestion
        log_print(LOG_WARNING, SECTION_FUSEDAV_STAT, "get_fresh_fd: unexpected response code: %u: ", response_code);
```
1) Fix typo ("Repsonse"), 2: alter log message to be less wordy/more similar to other fusedav log messages.

Q: Is this `log_print` necessary/useful, given the `g_set_error` call ("File ... expected to exist returns ...") two lines later?

Reply:
Deleted line 757 :)